for_attendees.md

```diff
@@ -1,120 +1,127 @@
-# Remote Learning Thoughts for Workshop Attendees
+# Remote Learning Instructions for Workshop Attendees
 
 ### Introduction
 
-This document is an attempt to list all many points that need to be considered when attending
- a class that offers a remote option or is taught entirely remotely.  As in-person support is
-  not available, it is imperative that proper planning and sufficient lead time are given to
-   ensure a smooth and enjoyable class experience. Additionally, we&#39;d like to encourage
-    as much interaction as possible and when reasonable.
+This document lists points to consider when attending a workshop that offers a remote option
+or is taught entirely remotely. Most importantly, **ensure that all required software is
+installed and working correctly ahead of time**. As in-person support is not available, it is
+imperative that proper planning and sufficient lead time are given to ensure a smooth and
+enjoyable class experience. Additionally, we&#39;d like to encourage as much interaction as
+possible and when reasonable.
 
```

Comment:
I tried to project a bit more confidence (e.g., s/thoughts/instructions/g) and bold the most important part for skimmers.